client/packages/requisitions/src/RequestRequisition/DetailView/DetailView.tsx

```diff
@@ -29,6 +30,7 @@ export const DetailView: FC = () => {
   const { setCustomBreadcrumbs } = useBreadcrumbs();
   const navigate = useNavigate();
   const { data, isLoading } = useRequest.document.get();
+  const { requisitionId = '' } = useParams();
```

Comment:
I thought better to get this from url params instead of waiting for data to return